lua/ulx/modules/sh/cfc_cleanup.lua

```diff
@@ -34,26 +34,39 @@ ropes:defaultAccess( ULib.ACCESS_ADMIN )
 ropes:help( "Remove target(s) ropes" )
 
 
-local function cleanupPlayerEnts( callingPlayer, targetPlayers )
+local function cleanupPlayerEnts( callingPlayer, targetEntities, targetPlayers )
     local isTarget = {}
     for _, ply in pairs( targetPlayers ) do
         isTarget[ply] = true
     end
     
     local count = 0
+
+    local isWildcardMatch = targetEntities == "*"
+
     for _, ent in ipairs( ents.GetAll() ) do
+        if ent:IsWeapon() then continue end
+
+        local isModelMatch = targetEntities == ent:GetModel()
+        local isClassMatch = targetEntities == ent:GetClass()
+        local isMatch = isWildcardMatch or isModelMatch or isClassMatch
+
+        if not isMatch then continue end
+
         local owner = ent.CPPIGetOwner and ent:CPPIGetOwner()
-        if isTarget[owner] and not ent:IsWeapon() then
-            ent:Remove()
-            count = count + 1
-        end
+
+        if not isTarget[owner] then continue end
+
+        ent:Remove()
+        count = count + 1
     end
 
     ulx.fancyLogAdmin( callingPlayer,  "#A removed "..count.." entities owned by #T", targetPlayers )
 
 end
 
 local cleanup = ulx.command( CATEGORY_NAME, "ulx cleanup", cleanupPlayerEnts, "!cleanup" )
+cleanup:addParam{ type=ULib.cmds.StringArg, hint="class/model, * for all" }
```

Comment:
maybe this could be the 2nd param, then if someone wanted to cleanup everything they could just exclude the model and type `ulx cleanup target` instead of `ulx cleanup * target`